src/types/user.go

```diff
@@ -11,12 +11,12 @@ type UserIdentifier struct {
 }
 
 type UserInput struct {
-	Namespace string `json:"namespace" binding:"required"`
+	Namespace string `json:"namespace" binding:"required,oneof=admin viewer contributor"`
```

Comment:
These are not the names of the roles we've created. `admin` already exists as a ClusterRole in Kubernetes. Should be `capp-admin`, `capp-viewer` and `capp-contributor`

Reply:
There's a map which converts this values in the controller I think for the platform it's much more clean and understands